src/EPL-2.0.xml

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+	<license isOsiApproved="true" licenseId="EPL-2.0"
+		name="Eclipse Public License 2.0">
+		<crossRefs>
+			<crossRef>http://www.eclipse.org/legal/epl-2.0</crossRef>
+			<crossRef>http://www.opensource.org/licenses/EPL-2.0</crossRef>
```

Comment:
This should use `https://opensource.org/licenses/EPL-2.0`, since that's the OSI's canonical location:

```console
$ curl -sLI http://www.opensource.org/licenses/EPL-2.0 | grep '^HTTP\|Location'
HTTP/1.1 301 Moved Permanently
Location: http://opensource.org/licenses/EPL-2.0
HTTP/1.1 301 Moved Permanently
Location: https://opensource.org/licenses/EPL-2.0
HTTP/1.1 200 OK
```

I'd like to use HTTPS for eclipse.org too (https://www.eclipse.org/legal/epl-2.0/), although they allow both HTTP and HTTPS without redirects.